src/chart/pie/PieView.ts

```diff
@@ -189,7 +189,7 @@ class PiePiece extends graphic.Sector {
         setLabelLineStyle(this, getLabelLineStatesModels(itemModel), {
```

Comment:
Use a `show` flag in `setLabelLineStyle` is not a good style. It makes this function more complex and obscure because it's not matching with the function name. 

Instead, we should use `labelLine.hide()` when `position` is not 'outside' and `labelLine` has been created explicitly here.